src/docs/asciidoc/AdminUI.asciidoc

```diff
@@ -179,3 +181,31 @@ image::images/spring-xd-admin-ui-jobs-step-execution-history.png[Step Execution
 
 On this screen, you can view various metrics associated with the selected step such as *duration*, *read counts*, *write counts* etc.
 
+[[adminui-analytics]]
+=== Analytics
+
+The _Analytics_ section of the admin UI provides dashboarding and data visualization capabilities for the various counter modules available in _Spring XD_:
+
+* Counters
+* Aggregate Counters
+* Field-Value Counters
+* Gauges
+* Rich Gauges
+
+For example, if you have created the `springtweets` stream and the corresponding counter in the <<counter, Counter chapter>>, you can now easily create the corresponding graph from within the **Dashboard** tab:
+
+1. Under `Metric Type`, select `Counters` from the select box
+2. Under `Stream`, select `tweetcount`
+3. Under `Visualization`, select the desired chart option, e.g. `Graph Chart`
+
+Afterwards you should see a visualization similar to the following:
+
+.Counter Graph
+image::images/spring-xd-admin-ui-analytics-counter.png[Analytics Counter, width=500]
```

Comment:
Since we only have Graph and Bar charts here, I think we can add `Graph chart` as well. I will add the content for it while merging.